playwright-tests/circuit.ts

```diff
@@ -0,0 +1,498 @@
+export const validCircuitJson = [
```

Comment:
not the best name here, `exampleCircuitJson` as an export and the name of the file would be much better. When in doubt the export should almost always match the file name.